README.md

```diff
@@ -1,37 +1,79 @@
+
+<details><summary markdown="span"><code>Fold out to see sample</code></summary>
+
+<img width="1680" alt="Screenshot CageManager" src="https://user-images.githubusercontent.com/1786772/43678602-914198ec-9816-11e8-8a75-0ac3032368e4.png">
+
+</details>
+
+### CageChooser
+
+The `CageChooser` is a user interface which displays in a list all available configs on the system by iterating over the registry entries under the following path: `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\SharkCage\Configs`. By selecting a config and pressing the "Start"-button (or the enter key), the `CageManager` will be started. As soon as the `CageManager` is started, the `CageChooser` will send the path to the selected config to the `CageManager`.
+
+<details><summary markdown="span"><code>Fold out to see sample</code></summary>
 
-### StarterCmd
+<img width="500" alt="Screenshot CageChooser" src="https://user-images.githubusercontent.com/1786772/43733677-c50f6b72-99b5-11e8-9831-69556fd33246.png">
 
-A user interface for starting the `CageManager` and an application in it. Directly communicating with the `CageService`.
+</details>
 
-### ImageSelectDialog
+### CageConfigurator
 
-A user interface to select an icon to be displayed on the user's desktop. `config.txt` will be created in `C:\sharkcage` with a line containing `picture: ` followed by the index of the icon. The `CageService` reads this configuration file and searches for that line and opens the dialog if the file contains no such line.
+The `CageConfigurator` is a user interface to create a config file including an icon to be displayed on the secure desktop, the program which should be started and an optionally additional program. The additional program can be choosen out of a list of "trustworthy" applications.
+ As soon as the config file has been saved, a link to the config which is stored under `C:\Users\Public\Documents\SharkCage\` will be saved in the Registry under the following path: `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\SharkCage\Configs`.
+ The config contains json data and is protected using ACL's to be read and edited only by the `CageConfigurator` in a secure environment, the `CageService` as well as the administrator group.
+
+<details><summary markdown="span"><code>Fold out to see sample</code></summary>
+
+<img width="1680" alt="Screenshot CageConfigurator" src="https://user-images.githubusercontent.com/1786772/43733731-e88c2e50-99b5-11e8-84a5-860351997f55.png">
+
+</details>
+
+### SharkCageInstaller
+
+The `SharkCageInstaller` is used to install all project applications, starts the `CageService` and sets some keys in the registry.
```

Comment:
'[...], starts the `CageService` and sets some keys in the registry.' -> '[...], starting the `CageService` and setting some keys in the registry.'